pkgs/development/python-modules/weasyprint/default.nix

```
@@ -0,0 +1,54 @@
+{ buildPythonPackage,
+  fetchPypi,
+  cairosvg,
+  pyphen,
+  cffi,
+  cssselect,
+  lxml,
+  html5lib,
+  tinycss,
+  pygobject2,
+  glib,
+  pango,
+  fontconfig,
+  stdenv,
+  pytestrunner,
+  isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "WeasyPrint";
```

```suggestion
  pname = "weasyprint";
```
You'll need to put `pname = "WeasyPrint";` inside `fetchPypi`.

---

Done